packages/vhd-lib/src/Vhd/VhdAbstract.integ.spec.js

```diff
@@ -0,0 +1,140 @@
+/* eslint-env jest */
+
+import rimraf from 'rimraf'
+import tmp from 'tmp'
+import fs from 'fs-extra'
+import { getSyncedHandler } from '@xen-orchestra/fs'
+import { Disposable, pFromCallback } from 'promise-toolbox'
+
+import { openVhd } from '../index'
+import { createRandomFile, convertFromRawToVhd, createRandomVhdDirectory } from '../tests/utils'
+import { VhdAbstract } from './VhdAbstract'
+
+let tempDir
+
+jest.setTimeout(60000)
+
+beforeEach(async () => {
+  tempDir = await pFromCallback(cb => tmp.dir(cb))
+})
+
+afterEach(async () => {
+  await pFromCallback(cb => rimraf(tempDir, cb))
+})
+
+test('It creates an alias', async () => {
+  await Disposable.use(async function* () {
+    const handler = yield getSyncedHandler({ url: 'file:///tmp/' })
```

Comment:
Use `os.tmpdir()`.

Reply:
Also, you can mutualize the synced between test by moving it to `before()`/`after()` but it's up to you.

Reply:
I will need tests across multiple remotes later. 
Also all the other tests use `tmp` . What are the advantage of not using it ?

Reply:
You don't know the system the tests are running on, the temporary directory might not be `/tmp`, use `os.tmpdir()` instead.

Reply:
Ok, let's go with this.
only on this test and the later one, or should I fix all the 15 tests suites using this ?